frontend/all.js

```diff
@@ -41,7 +41,8 @@ function initAll() {
 
   var $filters = document.querySelectorAll('[data-module="activities-list-filter"]')
   nodeListForEach($filters, function ($filter) {
-    new ListFilter($filter)
+    const startShown = $filter.getAttribute('data-filter-start-shown') === 'true'
```

Comment:
Extremely minor, but personally I'd prefer this logic in the `ListFilter` component code so all the component logic is together.

And you might prefer to use the element's `dataset` to get the value here.

Reply:
Completely correct this should be in `ListFilter` so I've moved it. I didn't even know about `dataset` so thanks for teaching me something :)